dll/win32/newdev/newdev.c

```diff
@@ -353,6 +354,10 @@ SearchDriverRecursive(
     BOOL retval = FALSE;
     HANDLE hFindFile = INVALID_HANDLE_VALUE;
 
+    /* Path + 3 characters (pattern + zero terminal) is too long to be searched */
+    if (wcslen(Path) + 3 > MAX_PATH)
```

Comment:
```suggestion
    /* Path with pattern is too long to be searched */
    if (wcslen(Path) + 2  >= _countof(PathWithPattern))
```
And adding `L"\\"` is conditional, so that should be accounted for here too.